src/App.tsx

```diff
@@ -1,61 +1,182 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { TodoList } from './components/TodoList/TodoList';
+
+import { User } from './type/User';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+function checkKey() {
+  const maxId = [...todos].sort((a, b) => b.id - a.id);
```

Comment:
```suggestion
  const maxId = [...todos].sort((a, b) => b.id - a.id)[0];
```
you mate take the max id, to not keep all the array